tests/utils/zmq_sender.py

```diff
@@ -13,7 +13,7 @@ def send(topic, msg, host="127.0.0.1", port=50000, bind=True):
     else:
         socket.connect(f"tcp://{host}:{port}")
         time.sleep(0.5)
-    print(f"send string: {topic} {msg}")
+    # print(f"send string: {topic} {msg}")
```

Comment:
Maybe a tracing/debug statement instead?

Reply:
There is no `logging` used in the test utils. Also, the output from the Python `logging` framework is hidden in unit tests.